Test_November12/ThreadSafePriorityQueue.cs

```diff
@@ -0,0 +1,53 @@
+namespace Test_November12;
+
+public class ThreadSafePriorityQueue<T, TP> where TP : IComparable
+{
+    private readonly SortedList<TP, Queue<T>> _priorityQueue = new SortedList<TP, Queue<T>>();
+    private int _size;
+    public void Enqueue(T value, TP priority)
```

Comment:
```suggestion

    public void Enqueue(T value, TP priority)
```
И отсутствуют summary для публичных методов и свойств.

Reply:
-0.5